src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -242,7 +242,7 @@ private function getDocBlockFromProperty($class, $property)
      * @param string $ucFirstProperty
      * @param int    $type
      *
-     * @return DocBlock|null
+     * @return array|null
```

Comment:
This change should be reverted (this method returns an instance of DocBlock, not an array).